Automation/create_vm_script.ps1

```diff
@@ -0,0 +1,68 @@
+#Script to create a Virtual Machine on Azure.
+#https://docs.microsoft.com/en-us/azure/virtual-machines/windows/quick-create-powershell
+
+#Note: This creates a Windows VM, but the end goal is a Linux VM. This will be the next sprint.
+#Note: There are extra parameters that are not being used yet. These will be used when creating Linux machine.
+param (
+    [string] [Parameter(Mandatory=$true)] $tenantId, #Tenant ID used to login to Azure with Service Provider
+    [string] [Parameter(Mandatory=$true)] $applicationId, #App ID used to login to Azure with Service Provider
+    [string] [Parameter(Mandatory=$true)] $secret, #Azure Secret used to login to Azure with Service Provider
+    [string] [Parameter(Mandatory=$true)] $subscriptionId, #Subscr. ID used to login to Azure with Service Provider
+    [string] [Parameter(Mandatory)] $rgname, #Resource Group Name required.
```

Comment:
can we change rgname and vmname to rgName and vmName? I got this script messed up because I assumed camelcase for all the variables. (When I changed it to lowercase, it worked fine)